### PR TITLE
Update TruckersFM Presence

### PR DESCRIPTION
**Pull request by pascalla**
- Updated Artwork
- Updated Description
- Updated Colour
- Updated ags

### CODE REVIEW

File: TruckersFM/dist/metadata.json [hunk 1]
```diff
@@ -1,23 +1,25 @@
 {
   "author": {
-    "name": "Ninja",
-    "id": "150987570715688960"
+    "name": "Pascal",
+    "id": "156560379089125376"
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: You shouldn't change the author, you can add yourself as a contributor

File: TruckersFM/dist/metadata.json [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
-    "en": "The number one hit music station for Trucking Simulators. Radio made for driving, now playing online, in cab and on the go."
+    "en": "Truckers.FM is the  No.1 Hit Music Station as voted by the TruckersMP Community 5 years running. Broadcasting live online, on TuneIn and on your smart speaker, Truckers.FM provides round the clock entertainment broadcasting a variety of shows, ranging from the Top 40 to the biggest throwbacks. Be sure to follow us on our social media channels @TruckersFM"
   },
-  "url": "truckers.fm",
-  "version": "1.0.2",
-  "logo": "https://ninjalabs.co.uk/i/p0ezmnp.png",
-  "thumbnail": "https://ninjalabs.co.uk/i/i48db0v.png",
-  "color": "#fc0303",
+  "url": "https://truckers.fm",
+  "version": "1.0.3",
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: url shouldnt include https://

File: TruckersFM/dist/metadata.json [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
+  "color": "#E1306C",
   "tags": [
     "TFM",
+    "TruckersMP",
     "radio",
+    "music",
     "ETS2",
     "ATS",
-    "Your Drive, Your Music"
+    "Simulator Radio"
```
Comment (Bas950) [inline TruckersFM/dist/metadata.json]: tags are not allowed to have spaces anymore use -

**Comment by pascalla** [comment]
> Also this only updates the metadata is that correct?

Yeah, we noticed the images were hosted on a self upload site by original creator which seems to have invalid certs, also took opportunity to have some nicer images :)

**Comment by Bas950** [comment]
> Yeah, we noticed the images were hosted on a self upload site by original creator which seems to have invalid certs, also took opportunity to have some nicer images :)

All good, regarding the other tags, the rule changed a week ago so I looked it up:
• Tags should be used as alternate names whenever possible, shortened versions must be included as well (e.g. if an Amazon presence had included AWS support it would have its tags like : "amazon-web-services" and "aws"). This is not required if it's not possible, but will make it easier for users when searching.
• Tags must not include any spaces, slashes, single/double quotation marks, unicode characters and should always be lowercase.

Could you update the other tags as well?

**Comment by pascalla** [comment]
> Make the description shorter ( x3 ) and remove the part where you promote your social media account.

Hope changes are satisfactory

**Pull request merged**